src/pages/TeamSettings/Service-Accounts.vue

```diff
@@ -41,9 +41,9 @@ export default {
   computed: {
     ...mapGetters('tenant', ['tenant']),
     ...mapGetters('user', ['user']),
-    ...mapGetters('license', ['license']),
+    ...mapGetters('license', ['license', 'hasPermission']),
     isTenantAdmin() {
-      return this.tenant.role === 'TENANT_ADMIN'
+      return this.hasPermission('create', 'role')
```

Comment:
I think this should be doing individual checks on create, update, and delete `*:service-account`